### DISCUSSION TITLE
How do I update the Quarto report (F5)?

### DISCUSSION THREAD

**Discussion by caiolivf** [quarto-dev/quarto-cli#6202] [Q&A]
### Description

Hello!

I'm developing a report in HTML format using RStudio version "2023.06.0 Build 421" and I would like to know if there is any way to render/update the report through the browser (something like pressing F5).

Any other alternative will be welcome.

**Comment by mcanouil** [comment]
No there isn't (and there very likely never will), use the preview option in RStudio which should update the rendered content on changes.
